ui/app/rif/components/modal/addNewTokenNetworkAddress/addNewTokenNetworkAddress.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Select from 'react-select'
+
+class AddNewTokenNetworkAddress extends Component {
+	static propTypes = {
+		updateNetwork: PropTypes.func.isRequired,
+		updateAddress: PropTypes.func.isRequired,
+		networks: PropTypes.array.isRequired,
+	}
+    constructor(props) {
+        super(props);
+		let networks = []
+		Object.assign(networks, props.networks)
+		this.state = { 
+            networks: networks,
+			selectedNetwork: networks[0],
+			insertedAddress: '',
+		};
+	}
+	_updateNetwork = (selectedOption) => {
+		this.setState({ selectedNetwork: selectedOption })
+		this.props.updateNetwork(selectedOption)
+	}
+	_updateAddress = (e) => {
+		this.setState({ insertedAddress: e.target.value })
+		this.props.updateAddress(e.target.value)
+	}
+	
+	render () {
+        const {  } = this.props
```

Comment:
Is this destructuring needed? You are not getting any attribute of props here

Reply:
I was using some props to testing, miss this code there! Thanks!